src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -590,6 +590,48 @@ module Types = struct
                    Option.map
                      ~f:(get_best_ledger_account coda)
                      account.Account.Poly.delegate )
+             ; field "delegatees"
```

Comment:
if a delegator is someone who is delegating to someone else, wouldn't the delegatee be the person that is being delegated to? It seems like this is a list of the delegators to this account, not its delegatees. That said, all this delegatation vocabulary is very confusing so I might be wrong @o1pranay

Reply:
my opinion:

delegator -- the one who delegates funds to a target (eg. the user who wants to delegate, but not stake directly)
delegate (noun) -- the one who is the target of delegated funds. eg. the one who runs the staking node
delegate (verb) -- the delegator delegates funds to the delegate
delegatee -- I think this is synonymous with delegate (noun) -- ie. the target

My suggestion is to use `delegator` to mean the source, `delegate` to mean the verb to delegate funds, and `target block producer` or `target` to mean the target -- this way we avoid namespacing that is too close to each other

Reply:
Thanks, I guess in the context of this PR, it would mean to rename delegatees -> delegators

Reply:
I changed the fields from `delegatees` to `delegators` as suggested.